scripts/Neopets_Bank_and_Till/main.user.js

```diff
@@ -39,7 +39,7 @@ function get_np(p, n) {
 
 // bank
 if (location.pathname == "/bank.phtml") {
-	GM_setValue("bank", get_np(document, "id('content')/table/tbody/tr/td[2]/div/table/tbody/tr[2]/td/table/tbody/tr[2]/td[2]"));
+	GM_setValue("bank", get_np(document, "id('content')/table/tbody/tr/td[2]/div[2]/table/tbody/tr/td[2]/table/tbody/tr[2]/td[2]"));
```

Comment:
I used a plugin called [XPath Checker](https://addons.mozilla.org/en-US/firefox/addon/xpath-checker/) to test your xpath and it didn't work.
`div[2]` worked but `tr/td[2]` instead of `tr[2]/td` broke it.
The script works the way it currently is.